partials/card.hbs

```diff
@@ -3,23 +3,23 @@ Re-usable card for linking to posts
 --}}
 
 <article class="gh-card {{post_class}} {{#if feature_image}}with-image{{else}}no-image{{/if}}">
-    <a class="gh-card-link" href="{{url}}">
+    <div class="row no-gutters border rounded overflow-hidden flex-md-row align-items-center mb-4 shadow-sm h-md-250 position-relative">
         {{#if feature_image}}
+        <div class="col-md d-block">
             <img class="gh-card-image"
                 srcset="{{img_url feature_image size="s"}} 300w,
-                        {{img_url feature_image size="m"}} 600w,
-                        {{img_url feature_image size="l"}} 1000w,
-                        {{img_url feature_image size="xl"}} 2000w"
+                    {{img_url feature_image size="m"}} 600w,
+                    {{img_url feature_image size="l"}} 1000w,
+                   {{img_url feature_image size="xl"}} 2000w"
```

Comment:
Please restore the original spacing for these three lines to ensure consistency.